remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyRemotingServer.java

```diff
@@ -206,7 +206,7 @@ public Thread newThread(Runnable r) {
                 .channel(useEpoll() ? EpollServerSocketChannel.class : NioServerSocketChannel.class)
                 .option(ChannelOption.SO_BACKLOG, 1024)
                 .option(ChannelOption.SO_REUSEADDR, true)
-                .option(ChannelOption.SO_KEEPALIVE, false)
+                .childOption(ChannelOption.SO_KEEPALIVE, false)
```

Comment:
The default value of childOption is true or false?

Reply:
Default tcp-ping is on. As we have application-level heartbeats, it is not required.